### DISCUSSION TITLE
Shift towards passwordless/passkey authentication.

### DISCUSSION THREAD

**Discussion by bradleyess** [ory/kratos#2662] [Ideas]
I've just discovered Hanko and although I'm skeptical of how simple this looks on the surface, I am particularly fond of the passcode login strategy. 

https://github.com/teamhanko/hanko

Forgive me if this is covered somewhere else, but is there a place for us to stay up to date with any efforts to add passcode logins to Kratos? I have found discussions pertaining to TOTP but Authenticator is often mentioned in the same sentence. While this is great, I believe there is value in the passcode approach for user bases with lower technical literacy.

**Comment by aeneasr** [comment]
Absolutely! PassCode is basically just WebAuthn, which we support for passwordless login. It’s probably just a matter of using the right configs!

**Comment by aeneasr** [comment]
Hey, I just checked. If you enable passkey support in Safari, you can use Pass Keys with Ory already. However, you need to enable Pass Key support in the developer settings as it is not yet fully released at Apple:


> First of all, enable Platform Authenticator Syncing on your Apple devices. In iOS 15, turn on the Syncing Platform Authenticator switch under Settings > Developer. The Developer menu is available on your device when you set it up as a development device in Xcode.
>
> In macOS Monterey, go to Safari > Preferences, click the Advanced tab, and select the “Show Develop menu in menu bar” option. Then enable the Develop > Enable Syncing Platform Authenticator menu item in Safari.

Here is an example project: https://pedantic-wright-03evpziv3q.projects.oryapis.com/ui/registration

And here the video - for me the last step failed because my iPhone does not have the XCode set up done yet.

https://user-images.githubusercontent.com/3372410/184266119-f824292d-15f4-41bd-b24b-7ce7287ac1b4.mov

Here are the settings i used

<img width="1661" alt="Screenshot 2022-08-11 at 18 01 32" src="https://user-images.githubusercontent.com/3372410/184266419-0ac80f1c-788d-4cd7-b652-5de428c1ade6.png">
<img width="1417" alt="Screenshot 2022-08-11 at 18 01 38" src="https://user-images.githubusercontent.com/3372410/184266410-835311b0-6b0a-4f60-9101-600491991736.png">
